<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Диспетчер запуска программ</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/process.png</normaloff>:/images/process.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="file_menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="settings_menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="autolaunch_action"/>
    <addaction name="save_output_action"/>
    <addaction name="separator"/>
    <addaction name="reread_taskslist_action"/>
   </widget>
   <addaction name="file_menu"/>
   <addaction name="settings_menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>toolbar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="autolaunch_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Автозапуск</string>
   </property>
   <property name="statusTip">
    <string>Автозапуск задач при старте программы</string>
   </property>
  </action>
  <action name="save_output_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Сохранять вывод</string>
   </property>
   <property name="statusTip">
    <string>Сохранение сообщений стандартного вывода</string>
   </property>
  </action>
  <action name="reread_taskslist_action">
   <property name="text">
    <string>Перечитать список задач</string>
   </property>
   <property name="statusTip">
    <string>Обновление списка задач</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
